docs/source/user/aerodyn/driver.rst

```diff
@@ -0,0 +1,497 @@
+When `DOF=0`, the turbine base is fixed.
+
+
+.. code::
+
+    ----- Time-dependent Analysis [used only when AnalysisType=3 and numTubines=1] ------
+             4  NumCases     - Number of cases to run
+    HWndSpeed  PLExp   RotSpd   Pitch   Yaw    dT      Tmax   DOF   Amplitude  Frequency 
+    (m/s)      (-)     (rpm)    (deg)  (deg)   (s)     (s)    (-)    (-)       (Hz)
+       8.      0.0       6.     0.      0.     1.0     100     0      0         0.0
+       8.      0.0       6.     0.      0.     1.0     100     0      0         0.0
+       9.      0.1       7.     1.      0.     0.5      50     1      5.0       0.1 
+       9.      0.2       8.     2.      0.     0.5      50     1      2.0       0.2 
+
+
+**Outputs**
+
+The output section controls the format of the tabular output file and VTK files, similar to the OpenFAST outputs.
+The user can control the hub radius and nacelle dimension for the VTK visualization. The nacelle is represented as a sphere of radius (`VTKHubRad`), and the nacelle with a parallelepiped defined using an origin and three lengths parallel to the nacelle coordinates (`VTKNacDim`).
+
+
+.. code::
+
+    ----- Output Settings -------------------------------------------------------------------
+      "ES15.8E2"     OutFmt      - Format used for text tabular output, excluding the time channel.  Resulting field should be 10 characters. (quoted string)
+    2                OutFileFmt  - Format for tabular (time-marching) output file (switch) {1: text file [<RootName>.out], 2: binary file [<RootName>.outb], 3: both}
+    0                WrVTK       - VTK visualization data output: (switch) {0=none; 1=animation}
+    2                VTKHubRad   - HubRadius for VTK visualization (m)
+    -1,-1,-1,2,2,2   VTKNacDim   - Nacelle Dimension for VTK visualization x0,y0,z0,Lx,Ly,Lz (m)
+
+
+
+
+
+.. _ad_inputs_multirot:
+
+AeroDyn inputs for multiple turbines
+------------------------------------
+
+No changes are required to the AeroDyn input files when one turbine is used. 
+To minimize the impact of the multiple-turbines implementation, the driver currently uses only one AeroDyn input file for all turbines. 
+This means that the AeroDyn options are currently the same for all rotors.
+
+The definition of the blade files and tower inputs needs to be adapted when more than three blades are used and more than one turbine is used.
+
+**Blade files**
+
+The legacy AeroDyn format requires a minimum of three blade file names. 
+For this reason, the blades of all rotors are currently indicated successively in the `ADBlFile` list. 
+The list is populated by looping on turbines and turbine blades, with tine blades being the fast index.
```

Comment:
What is "tine"?  Suggest rewording.

Reply:
I'm not sure what happened there.. The blade index is the fastest index.